biojava-aa-prop/src/main/java/org/biojava/nbio/aaproperties/CommandPrompt.java

```diff
@@ -327,9 +327,7 @@ private static void compute(PrintStream output, String header, String sequence,
 			}
 		}
 		output.print(header.replace(delimiter, "_"));
-		for(int i = 0; i < dList.size(); i++){
-			output.print(delimiter + Utils.roundToDecimals(dList.get(i), decimalPlace));
-		}
+		dList.stream().forEach(item -> System.out.print(delimiter + Utils.roundToDecimals(item, decimalPlace)));
```

Comment:
This is not the same. The original code was printing to `output` and not to `System.out`